wagtail/project_template/home/tests.py

```diff
@@ -0,0 +1,39 @@
+# home/tests.py
```

Comment:
```suggestion
```

Do we need this comment? it does not look like we do this in any other template files.